packages/core/components/Modal/CopyFileManifest/index.tsx

```diff
@@ -11,6 +11,19 @@ import { interaction, selection } from "../../../state";
 
 import styles from "./CopyFileManifest.module.css";
 
+function ResponsiveFileName({ fileName }: { fileName: string }) {
+    const lastDot = fileName.lastIndexOf(".");
+    const base = lastDot > 0 ? fileName.slice(0, lastDot) : fileName;
+    const extension = lastDot > 0 ? fileName.slice(lastDot) : "";
```

Comment:
this will not account for file types like .ome.zarr, although that may not be a major concern

Reply:
or other files that have a tag like _outlines.ext that are derivitive. I dont think its a big deal.